app/models/invitation.rb

```diff
@@ -26,6 +26,7 @@ class InvitationAlreadyUsed < StandardError
   scope :pending, -> { not_cancelled.single_use.where(accepted_at: nil) }
   scope :shareable, -> { not_cancelled.where(single_use: false) }
   scope :single_use, -> { not_cancelled.where(single_use: true) }
+  scope :ignored, -> (send_count, age_in_hours) { pending.where('send_count = ? AND created_at < ?', send_count, age_in_hours.hours.ago) }
```

Comment:
I'd also slightly prefer, instead of `age_in_hours`, `since`

```
-> (send_count, since) { pending.where(sc: sc).where('created_at < ?', since) }
```

which takes out the slight magical number-ness of
```
resend_ignored(send_count: 1, age_in_hours: 24)
resend_ignored(send_count: 2, age_in_hours: 72)
```

and instead reads

```
resend_ignored(send_count: 1, since: 1.day.ago)
resend_ignored(send_count: 2, since: 3.days.ago)
```
?
💛